lgn-messages/src/types/v1/preprocessing/ext_tasks.rs

```diff
@@ -465,13 +465,30 @@ impl From<ExtractionRow> for TableRow {
 /// Inputs for an off-chain extraction.
 #[derive(Clone, Dbg, PartialEq, Deserialize, Serialize)]
 pub struct OffchainExtraction {
+    /// Determines which table this extraction corresponds to.
     pub table_id: TableId,
+
+    /// The "block_nr" of this extraction, also called `revision`.
     pub block_nr: BlockNr,
-    pub is_first_block: bool,
```

Comment:
A `bool` is not sufficient because the previous revision is not necessarily `block_nr-1`